rdf_entity.services.yml

```diff
@@ -38,7 +38,16 @@ services:
     arguments: ['@current_route_match']
     tags:
       - { name: 'context_provider' }
-  rdf_entity.inbound_outbound_value_subscriber:
-    class: Drupal\rdf_entity\EventSubscriber\InboundOutboundValueSubscriber
+  rdf_entity.inbound_datetime_value_subscriber:
+    class: Drupal\rdf_entity\EventSubscriber\InboundValueDateTimeSubscriber
+    tags:
+      - { name: event_subscriber }
+  rdf_entity.outbound_value_subscriber:
```

Comment:
This can probably be rdf_entity.outbound_datetime_value_subscriber to match the inbound subscriber above.

Reply:
Oh yes, I was in a hurry